lua/nvim-lsp-installer/servers/pylsp/README.md

```diff
@@ -0,0 +1,12 @@
+# Pylsp
+
+## Installing pylsp plugins
+Pylsp has [third party plugins](https://github.com/python-lsp/python-lsp-server#3rd-party-plugins) which are not installed by default.
+
+If you want to install them using the same installation of pylsp as nvim-lsp-installer, you may run
+```vim
+:PylspInstall <list of python packages>
+```
+
+PS:
+pylsp is installed in a python venv located in `stdpath('data')/lsp_servers/pylsp`
```

Comment:
Do you think is needed for any particular reason? I think it might be a bit overkill (also, since the installation directory is customizable this isn't entirely true)

```suggestion
```

Reply:
At least for mypy, we might want to know from where to execute some commands:
[from mypy docs](https://mypy.readthedocs.io/en/stable/running_mypy.html#missing-imports) you have to run the binary to install some missing type stubs.

Reply:
Ah I see. It seems like you can also install the `types-PyYAML` pip package also, so that is at least easy via `:PylspInstall` :+1:.

If it turns out to be use cases around accessing a shell with the venv activated, we can consider another command that opens a new terminal window with the venv activated for you, like:

```vim
command! PylspVenv exe 'tabnew | term' | startinsert | call feedkeys("source ~/.local/share/nvim/lsp_servers/pylsp/venv/bin/activate^M")
```